src/main/java/net/sf/jabref/gui/entryeditor/EntryEditorTab.java

```diff
@@ -300,9 +281,7 @@ public FieldEditor getActive() {
 
     public void activate() {
         if (activeField != null) {
-            /**
-             * Corrected to fix [ 1594169 ] Entry editor: navigation between panels
-             */
+            // Corrected to fix [ 1594169 ] Entry editor: navigation between panels
```

Comment:
This can be also deleted I think. Or you could replace it to not point to the bug.